src/Chapter1.hs

```diff
@@ -478,7 +491,9 @@ Implement a function that returns the last digit of a given number.
   whether it works for you!
 -}
 -- DON'T FORGET TO SPECIFY THE TYPE IN HERE
-lastDigit n = error "lastDigit: Not implemented!"
+lastDigit :: Int -> Int
+lastDigit n = mod n' 10
+  where n' = if n < 0 then (-n) else n
```

Comment:
You can use the standard function `abs` to implement `lastDigit` more elegantly.
But nice work on using variables in one of the first tasks!